samples/automl/automlVisionModel.js

```diff
@@ -0,0 +1,607 @@
+
+  // Get the full path of the model evaluation.
+  const modelEvaluationFullId = client.modelEvaluationPath(
+    projectId,
+    computeRegion,
+    modelId,
+    modelEvaluationId
+  );
+
+  // Get complete detail of the model evaluation.
+  client
+    .getModelEvaluation({name: modelEvaluationFullId})
+    .then(responses => {
+      const response = responses[0];
+      console.log(util.inspect(response, false, null));
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_getModelEvaluation]
+}
+
+function displayEvaluation(projectId, computeRegion, modelId, filter_) {
+  // [START automl_vision_displayEvaluation]
+  const automl = require(`@google-cloud/automl`);
+  const math = require(`mathjs`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const modelId = `id of the model, e.g. “ICN12345”`;
+  // const filter_ = `filter expressions, must specify field, e.g. “imageClassificationModelMetadata:*”`;
+
+  // Get the full path of the model.
+  const modelFullId = client.modelPath(projectId, computeRegion, modelId);
+
+  // List all the model evaluations in the model by applying filter.
+  client
+    .listModelEvaluations({parent: modelFullId, filter: filter_})
+    .then(respond => {
+      const response = respond[0];
+      for (let element of response) {
+        // There is evaluation for each class in a model and for overall model.
+        // Get only the evaluation of overall model.
+        if (!element.annotationSpecId) {
+          const modelEvaluationId = element.name.split(`/`).pop(-1);
+
+          // Resource name for the model evaluation.
+          const modelEvaluationFullId = client.modelEvaluationPath(
+            projectId,
+            computeRegion,
+            modelId,
+            modelEvaluationId
+          );
+
+          // Get a model evaluation.
+          client
+            .getModelEvaluation({name: modelEvaluationFullId})
+            .then(responses => {
+              const modelEvaluation = responses[0];
+
+              const classMetrics =
+                modelEvaluation.classificationEvaluationMetrics;
+
+              const confidenceMetricsEntries =
+                classMetrics.confidenceMetricsEntry;
+
+              // Showing model score based on threshold of 0.5
+              for (let confidenceMetricsEntry of confidenceMetricsEntries) {
+                if (confidenceMetricsEntry.confidenceThreshold === 0.5) {
+                  console.log(
+                    `Precision and recall are based on a score threshold of 0.5`
+                  );
+                  console.log(
+                    `Model Precision: %`,
+                    math.round(confidenceMetricsEntry.precision * 100, 2)

```

Comment:
Two things:
1) Can this `Math.round` logic go into a helper function? (Perhaps you can declare it on what is now line 406, within the body of this function?)
2) Can we use a shorter name than `confidenceMetricsEntry`/`confidenceMetricsEntries`? Maybe just `entry` and `entries`?

cc @happyhuman - what are your thoughts?